source/includes/kubernetes/_k8_persistentvolumes.md

```diff
@@ -178,6 +178,77 @@ Retrieve a persistent volume and all its info in a given [environment](#administ
+
+```json
+{
+  "taskId": "1542bd45-4732-419b-87b6-4ea6ec695c2b",
+  "taskStatus": "PENDING"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/persistentvolumes</code>
+
+Create a persistent volume in a given [environment](#administration-environments).
+
+| Required Attributes                             | &nbsp;                                                                                                                                                                                                          |
+| ----------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `apiVersion` <br/>_string_                      | The api version (versioned schema) of the persistent volume.                                                                                                                                               |
+| `metadata` <br/>_object_                        | The metadata of the persistent volume.                                                                                                                                                                     |
+| `metadata.name` <br/>_string_                   | The name of the persistent volume.                                                                                                                                                                         |
+| `spec` <br/>_object_                            | The spec for the persistent volume.                                                                                                                                                                        |
+| `spec.accessModes` <br/>_array_                 | A list of access modes, the options are: ReadWriteOnce, ReadOnlyMany and ReadWriteMany.                                                                                                                          |
+| `spec.capacity.storage` <br/>_string_           | Measured in bytes. You can express storage as a plain integer or as a fixed-point integer using one of these suffixes: E, P, T, G, M, K. You can also use the power-of-two equivalents: Ei, Pi, Ti, Gi, Mi, Ki. |
+| `spec.hostPath.path` <br/>_string_              | Directory location on host. |
+
+| Optional Attributes                   | &nbsp;                                                                                        |
+| ------------------------------------- | --------------------------------------------------------------------------------------------- |
+| `kind`<br/>_string_                   | The string value representing the REST resource this object represents.                       |
```

Comment:
Isn't that supposed to be persistent volume?

Reply:
I think we just use this description for every create entity.